apps/playground/src/app/components/ag-grid/edit-complex-cells/edit-complex-cells-in-modal.component.ts

```diff
@@ -32,6 +32,7 @@ export class EditComplexCellsInModalModalComponent {
 }
 
 @Component({
+  selector: 'app-edit-complex-cells-in-modal-modal-grid',
```

Comment:
There are 4 of these. These were throwing a ton of errors in the console every time the playground was initialized. I just cleaned them up while trying to ensure all my stuff was working.